fs/src/writer_file.rs

```diff
@@ -39,6 +42,11 @@ impl FileWriter for LocalFile {
         }
 
         file_options.open(path)?.write_all(content.as_bytes())?;
+
+        #[cfg(target_family = "windows")]
+        crate::win_permissions::set_file_permissions_for_administrator(path)
+            .map_err(|err| WriteError::GenericError(err.to_string()))?;
+
```

Comment:
Could we document the permissions for both unix and windows (in a doc-comment for the method)? The behavior is a bit different.